### PR TITLE
Update Post Relationships

### LINKED ISSUE #7: create post API
- [ ] Add functionality to add images
- [ ]  Pass user_id as foreign key

### PR DESCRIPTION
**Pull request by aitikgupta**
Only Creating `/post/create` a post will not require a user id, all other endpoints will need an ID due to a one-to-many relationship.

Partially resolves https://github.com/Open-Sourced-Olaf/SocioMark/issues/7

### CODE REVIEW

File: backend/app/server/controllers/user.py [hunk 1]
```diff
@@ -70,3 +73,14 @@ async def delete_user(email: str):
     if user:
         await users_collection.delete_one({"email": email})
         return True
+
+
+# Add a post id to the user model
+async def add_post_id(user_id: str, post_id: str):
+    print("author:", user_id, "post:", post_id)
```
Comment (bodhisha) [inline backend/app/server/controllers/user.py]: Can remove the print statement right..

File: backend/app/server/routes/post.py [hunk 2]
```diff
@@ -17,10 +17,11 @@
 
 router = APIRouter()
 
+
 @router.post("/create", response_description="Post added into the database")
```
Comment (bodhisha) [inline backend/app/server/routes/post.py]: Now the route is returning status code 200 which is "OK". It is more preferable to return status code 201 which indicates successfully created

**Review by bodhisha — CHANGES_REQUESTED** [review]

File: backend/app/server/controllers/user.py (hunk 1 above)
Comment (aitikgupta) [reply backend/app/server/controllers/user.py]: woops. will fix XD

File: backend/app/server/routes/post.py (hunk 2 above)
Comment (aitikgupta) [reply backend/app/server/routes/post.py]: Makes sense, will update

File: backend/app/server/controllers/user.py (hunk 1 above)
Comment (aitikgupta) [reply backend/app/server/controllers/user.py]: Done

File: backend/app/server/routes/post.py (hunk 2 above)
Comment (aitikgupta) [reply backend/app/server/routes/post.py]: Done 👍🏼

**Review by DebugAgrawal — APPROVED** [review]

**Review by DebugAgrawal — APPROVED** [review]

**Review by bodhisha — COMMENTED** [review]
LGTM

**Pull request merged**